INSTALL.md

```diff
@@ -11,24 +11,45 @@
 
     Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Clean install
+## Prerequisite
 
-1. You can get eZ Platform using composer with the following commands:
+  These instructions assume you have strong technical knowledge and have already installed PHP, web server & a database server with a corresponding clean database needed for this software.
+  For further information on requirements see: https://doc.ez.no/display/EZP/Requirements
+
+## Install
+
+1. **Get eZ Platform**:
+
+    A. **Archive** (tar/zip)
+
+       Extract the eZ Platform 2015.01(or higher) archive to a directory, then execute post install scripts:
+
+       *Note: The post install scripts will ask you to fill in some settings, including database settings.*
+
+       ```bash
+       cd /<directory>/
+       curl -s http://getcomposer.org/installer | php
+       php -d memory_limit=-1 composer.phar run-script post-install-cmd
+       ```
+
+
+    B. **Composer**
+
+     You can get eZ Platform using composer with the following commands:
+
+     *Note: composer will take its time to download all libraries and when done you will be asked to fill in some settings, including database settings.*
 
        ```bash
        curl -s http://getcomposer.org/installer | php
        php -d memory_limit=-1 composer.phar create-project --no-dev --prefer-dist ezsystems/ezpublish-community <directory> <version>
        cd /<directory>/
        ```
-       
-      Options:
-      - `<version>`: `dev-master` to get current development version (pre release), `v2015.01` to pick a specific release, otherwise skip it to get latest stable release.
-      - For core development change '--prefer-dist' to '--prefer-source' to get full git clones, and remove '--no-dev' to get things like phpunit and behat installed.
 
-  After running command above, composer will take its time to download all libraries, then a wizard will ask you for setting to database and lastly a welcome screen should grant you instructions for finishing installation.
+     Options:
+       - `<version>`: `dev-master` to get current development version (pre release), `v2015.01` to pick a specific release, otherwise skip it to get latest stable release.
+       - For core development change '--prefer-dist' to '--prefer-source' to get full git clones, and remove '--no-dev' to get things like phpunit and behat installed.
 
-
-2. Setup folder rights **For *NIX users**:
+2. *Only for *NIX users* **Setup folder rights**:
```

Comment:
If we want to simplify instructions, maybe we should simplify this particular chapter... I know permissions are critical here, but it still takes a LOT of space in the instructions, while it's clearly not the main part.

Mention it, and move it to the bottom ? Maybe like "PHP needs to be allowed to write to the log and cache folders". More advanced instructions can be found at https://doc.ez.no/display/EZP6/Installing+eZ+Publish+on+a+Linux-UNIX+based+system?focusedCommentId=25264322#InstallingeZPublishonaLinux-UNIXbasedsystem-SettingupfolderpermissionYellow5.x." ( (short URL needed for the link ?)